KWZP2019/KWZP2019/AddEmployee.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace KWZP2019
+{
+    public partial class AddEmployee : Form
+    {
+        public AddEmployee()
+        {
+            InitializeComponent();
+            Clear();
+        }
+
+        private void label5_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void AddEmployee_Load(object sender, EventArgs e)
+        {
+
+        }
+        //Function which clears all text boxes
```

Comment:
- zamiast komentarza niech mówi to nazwa funkcji
- nazwy metod zaczynamy od małej litery